<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Confirmation Procedure</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TITLE">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1021</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>End procedure?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="no_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover:!pressed
{
  background-color: rgb(195,30,60);
}

QPushButton
{
  background-color: white;
color: black;
border-style: outset;
border-width: 3px;
border-radius:10px;
border-color: black;
}
</string>
    </property>
    <property name="text">
     <string>Restart
Procedure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yes_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover:!pressed
{
  background-color: rgb(195,30,60);
}

QPushButton
{
  background-color: white;
color: black;
border-style: outset;
border-width: 3px;
border-radius:10px;
border-color: black;
}
</string>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>600</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pictures/program_pictures/main_wallpaper.png</pixmap>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>TITLE</zorder>
   <zorder>no_button</zorder>
   <zorder>yes_button</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
